packages/config/config/devices/0x027a/zen25.json

```diff
@@ -292,6 +292,40 @@
 			]
 		}
 	},
+	"18": {
+		"label": "Enable/Disable Power Reports",
+		"description": "Enable or disable power reporting (Requires Firmware 2.0)",
```

Comment:
This should be modeled as a conditional parameter instead of the description: https://zwave-js.github.io/node-zwave-js/#/config-files/conditional-settings

Reply:
Awesome, I didn't know that. I pushed 5af7044 that I think resolves this. Thank you!